src/components/timeline/TimelineMarkerCanvas.js

```diff
@@ -205,7 +239,7 @@ class TimelineMarkerCanvas extends PureComponent {
     // Draw separators
     ctx.fillStyle = '#eee';
     for (let rowIndex = startRow; rowIndex < endRow; rowIndex++) {
-      const y = (rowIndex + 1) * rowHeight - viewportTop;
+      const y = (rowIndex + 1) * rowHeight - viewportTop - 1;
```

Comment:
"-1" because the top border is actually out of the canvas.

Let's take the example of rowHeight == 4, here is how the lines are laid out:

```
------ Top border (out of canvas, drawn in CSS)
pixel 0
pixel 1
pixel 2
------ pixel 3
pixel 4
pixel 5
pixel 6
------ pixel 7
pixel 8
pixel 9
pixel 10
------ bottom border (out of canvas, drawn in CSS)
```

We can see the first border is at 3, second border at 7. We really need to take 1 off the value.